### DISCUSSION TITLE
Wix5 Bundle installer show Blank Help Page

### DISCUSSION THREAD

**Discussion by kumarashokd** [wixtoolset/issues#8604] [WiX Development]
Wix : 5.0.0

Git Repo : https://github.com/kumarashokd/Wix5-Installer/tree/main/wix-bundle-installer

Steps:
1. checkout wix-bundle-installer
2. Run build-bundle.bat (Build demo-bundle and launch Help Page) 

Actual :
![image](https://github.com/wixtoolset/issues/assets/79136806/f1deb4c4-2dda-482e-8656-18cebbab4817)

Expected : 

![image](https://github.com/wixtoolset/issues/assets/79136806/c192c377-2a12-4803-a37d-455671249c8c)

thm.xml file has proper content but its not rendering in the installer UI  

        <Page Name="Help">
            <Label X="11" Y="80" Width="-11" Height="30" FontId="2" DisablePrefix="yes">#(loc.HelpHeader)</Label>
            <Label X="11" Y="112" Width="-11" Height="-35" FontId="3" DisablePrefix="yes">#(loc.HelpText)</Label>
            <Button Name="HelpCloseButton" X="-11" Y="-11" Width="75" Height="23" TabStop="yes" FontId="0">
                <Text>#(loc.HelpCloseButton)</Text>
                <CloseWindowAction />
            </Button>
        </Page>

**Comment by cz-dev-ge** [comment]
see https://github.com/wixtoolset/issues/issues/8808
